web/modules/custom/nlc_prototype/js/nlc_prototype.google_analytics.js

```diff
@@ -42,6 +42,14 @@
           }
         );
       }
+
+      if (typeof settings.path.currentPathIsSelfProfile !== 'undefined' && settings.path.currentPathIsSelfProfile === true) {
+        gtag('event', 'PageView', {
+          event_category: 'Own profile page',
+          event_label: '',
```

Comment:
From the docs it looks like we don't have to set event_label at all if it's blank